specs/bug6Spec.js

```diff
@@ -17,12 +17,17 @@ describe('bug 6', function() {
         });
         it('when flex-shrink is set explicitly to zero', function(done) {
             var css = 'div{flex: 1 0 0%;}';
-            var output = 'div{flex: 1 0;}';
+            var output = 'div{flex-grow: 1;flex-shrink: 0;flex-basis: 0;}';
```

Comment:
Like I said, I can't remember why this should be expanded, same applies to the rest of your changes…